java2typescript-jackson/src/main/java/java2typescript/jackson/module/writer/DeclaredModuleFormatWriter.java

```diff
@@ -0,0 +1,25 @@
+package java2typescript.jackson.module.writer;
+
+import static java.lang.String.format;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import java2typescript.jackson.module.grammar.Module;
+
+/**
+ * Generates TypeScript type definitions for given module in declared module format
+ */
+public class DeclaredModuleFormatWriter extends ExternalModuleFormatWriter {
```

Comment:
If I've understood correctly, then the output format generated by this class is called [ambient module](https://www.typescriptlang.org/docs/handbook/modules.html#ambient-modules). In this case I'd prefer the name AmbientModuleFormatWriter. 

FYI: Related topic: naming of InternalModuleFormatWriter and ExternalModuleFormatWriter was based on former pre- TypeScript 1.5 terminology (since TS 1.5 they are named differently: "internal modules" are now called "namespaces" and external modules are called just "modules").